Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/Definitions/BlobData.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Frends.AzureBlobStorage.ListBlobsInContainer.Definitions
+{
+    /// <summary>
+    /// Blob's data.
+    /// </summary>
+    public class BlobData
+    {
+        /// <summary>
+        /// Flat or hierarchically listing structure.
+        /// </summary>
+        /// <example>Flat</example>
+        public string ListingStructure { get; set; }
```

Comment:
I don't think this property is needed, since user can select which listing structure he wants to use.